rules/linux/execution_shell_via_background_process.toml

```diff
@@ -48,7 +48,8 @@ For more details on Elastic Defend refer to the [helper guide](https://www.elast
 """
 severity = "medium"
 timestamp_override = "event.ingested"
-tags = ["Domain: Endpoint",
+tags = [
```

Comment:
I'd vote to roll back this change. While minor, it will bump the rule version and be received as a major rule update since version bump +1.